recipe/recipe_test.go

```diff
@@ -85,10 +85,10 @@ func TestGetRecipe(t *testing.T) {
 
 		Convey("Then whe GetRecipe is called, one GET /recipes/ID call is performed and the expected error is returned", func() {
 			recipe, err := recipeClient.GetRecipe(ctx, testUserAuthToken, testServiceToken, recipeID)
-			So(err, ShouldResemble, &Error{
-				err:        errors.New("failed to get response from Recipe API: recipe API error"),
-				statusCode: http.StatusInternalServerError,
-			})
+			So(err, ShouldResemble, dperrors.New(
+				errors.New("failed to get response from Recipe API: recipe API error"),
+				http.StatusInternalServerError,
+				nil))
```

Comment:
bracket should be closed on the next line (keep in line with where scope was opened)